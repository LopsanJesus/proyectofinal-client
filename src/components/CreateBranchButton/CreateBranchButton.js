import React from 'react';
import { Card } from 'react-bootstrap';
import { PlusCircle } from 'react-bootstrap-icons';
import { useTranslation } from 'react-i18next';
import { Link } from 'react-router-dom';
import './CreateBranchButton.scss';

const CreateBranchButton = ({ treeId }) => {
  const { t } = useTranslation();
  return (
    <Link to={"/create-branch/" + treeId}>
      <Card className="createTreeButton">
        <Card.Body>
          <PlusCircle size={30} className="plusIcon" />
          <Card.Title>{t('branch.createNewBranch')}</Card.Title>
        </Card.Body>
      </Card>
    </Link>
  );
};

export default CreateBranchButton;
