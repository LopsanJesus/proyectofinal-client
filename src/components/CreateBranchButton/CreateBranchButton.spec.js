import React from "react";
import { shallow } from "enzyme";
import CreateBranchButton from "./CreateBranchButton";

describe("CreateBranchButton", () => {
  test("matches snapshot", () => {
    const wrapper = shallow(<CreateBranchButton />);
    expect(wrapper).toMatchSnapshot();
  });
});
