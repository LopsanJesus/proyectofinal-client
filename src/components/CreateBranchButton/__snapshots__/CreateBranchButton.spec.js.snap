// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateBranchButton matches snapshot 1`] = `ShallowWrapper {}`;
